drivers/lsm6dsxx/lsm6dsxx_saul.c

```diff
@@ -1,31 +1,10 @@
-/*
- * Copyright (C) 2017 OTA keys S.A.
- *
- * This file is subject to the terms and conditions of the GNU Lesser
- * General Public License v2.1. See the file LICENSE in the top level
- * directory for more details.
- *
- */
 
-/**
- * @ingroup     drivers_lsm6dsl
- * @{
- *
- * @file
- * @brief       SAUL implementation for the LSM6DSL 3D accelerometer/gyroscope.
- *
- * @author      Vincent Dupont <vincent@otakeys.com>
- * @author      Sebastian Meiling <s@mlng.net>
- *
- * @}
- */
-
-#include "lsm6dsl.h"
```

Comment:
We should leave the info in here and add your @author and a `Copyright (c) 2019 HAW Hamburg`